robolectric/src/test/java/org/robolectric/plugins/DefaultSdkPickerTest.java

```diff
@@ -131,15 +131,12 @@ public void withTargetSdkLessThanMinSdk_shouldThrowError() throws Exception {
   }
 
   @Test
-  public void withTargetSdkGreaterThanMaxSdk_shouldThrowError() throws Exception {
+  public void withTargetSdkGreaterThanMaxSdk_shouldUseMaxSdk() throws Exception {
     when(usesSdk.getMaxSdkVersion()).thenReturn(21);
     when(usesSdk.getTargetSdkVersion()).thenReturn(22);
-    try {
-      sdkPicker.selectSdks(buildConfig(new Config.Builder()), usesSdk);
-      fail();
-    } catch (IllegalArgumentException e) {
-      assertThat(e).hasMessageThat().contains("Package targetSdkVersion=22 > maxSdkVersion=21");
-    }
+    sdkPicker.selectSdks(buildConfig(new Config.Builder()), usesSdk);
+    assertThat(sdkPicker.selectSdks(buildConfig(new Config.Builder()), usesSdk))
+            .containsExactly(sdkCollection.getSdk(21));
```

Comment:
```suggestion
       .containsExactly(sdkCollection.getSdk(21));
```

https://github.com/robolectric/robolectric/actions/runs/11425517206/job/31787530599?pr=9680#step:5:29